rule.go

```diff
@@ -914,3 +911,14 @@ func (r *Rule) HasVar(s string) bool {
 	}
 	return false
 }
+
+//getSidMgs returns a string representing a sidmsg.map entry
+func (r *Rule) getSidMgs(s string) string {	
+	var sidmsg = fmt.Sprintf("%s || %s", string(r.SID), r.Description);
+	for _, ref := range r.References {
+		if ref.Type == "url" {
```

Comment:
Why restrict this to URL? All reference types are allowed in the sid-msg.map file, aren't they?

Reply:
you are correct, should be solved.